apps/web/components/ui/form/Schedule.tsx

```diff
@@ -86,8 +88,9 @@ const TimeRangeField = ({ name }: TimeRangeFieldProps) => {
               options={options}
               onFocus={() => setOptions(timeOptions())}
               onBlur={() => setOptions([])}
-              defaultValue={getOption(value)}
+              defaultValue={"getOption(value)"}
               onChange={(option) => {
+                console.log("@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@");
```

Comment:
Minor bits of cleanup left, otherwise looks good - I still need to test locally

Reply:
OMG, so ashamed about that. I always triple-check for that. 
Thanks for the extra pair of eyes!

Reply:
Fixed!